addons/udes_stock/models/stock_quant.py

```diff
@@ -101,3 +101,15 @@ def create_picking(self, picking_type, only_available=False, **kwargs):
         return Picking.with_context(quant_ids=self.ids).create_picking(
             picking_type, products_info, **kwargs
         )
+
+    @api.model
+    def get_available_quantity(self, product, locations):
+        """Get available quantity of product_id within locations."""
+        product.ensure_one()
+        domain = [("product_id", "=", product.id), ("location_id", "child_of", locations.ids)]
+        quants = self.search(domain).with_context(prefetch_fields=False)
```

Comment:
Nit: `Quant.search()` please

Reply:
How do we get `Quant` here?  Do we need to do `Quant = self.env[...]`?  Or use the class name (`StockQuant`)?

Reply:
hmm, seems we have to go down the env route...

Reply:
Changed, please review.